specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -4419,6 +4668,112 @@
         }
       }
     },
+    "RouteTableProperties": {
+      "properties": {
+        "routes": {
+          "type": "array",
+          "description": "List of all routes.",
+          "items": {
+            "$ref": "#/definitions/Route"
+          }
+        },
+        "labels": {
+          "type": "array",
+          "description": "List of all connections attached to this route table.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "associatedConnections": {
+          "type": "array",
+          "description": "List of all connections associated with this route table.",
+          "items": {
+            "$ref": "#/definitions/HubVirtualNetworkConnection"
+          }
+        },
+        "propagatingConnections": {
+          "type": "array",
+          "description": "List of all connections that advertise to this route table.",
+          "items": {
+            "$ref": "#/defintions/HubVirtualNetworkConnection"
```

Comment:
@lirenhe/ @number213 : can i make this ref be any of 2-3 existing resources and not just a single one ?